docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+﻿version: "3.3"
+services:
+    traefik:
+        image: "traefik:v2.10"
+        container_name: "Epsilon-Traefik"
+        command:
+            - "--log.level=DEBUG"
+            - "--api.insecure=true"
+            - "--providers.docker=true"
+            - "--providers.docker.exposedbydefault=false"
+            - "--entrypoints.web.address=:80"
+            - "--entrypoints.websecure.address=:443"
+            # HTTPS / Certificates
+            - "--certificatesresolvers.letsencrypt.acme.email=k.janssen-ab@student.fontys.nl"
```

Comment:
Personal email

Reply:
Sure? What would you like it to be? :)

Reply:
Someone who is responsible, I think for now we can make it `typiqally@gmail.com` since the project is under my name. But in the future we could also make it smth like `epsilon@deltafhict.nl`, something for the future though!

Reply:
changed